test/regtest_joinmarket.cfg

```diff
@@ -14,15 +14,25 @@ rpc_user = bitcoinrpc
 rpc_password = 123456abcdef
 network = testnet
 
-[MESSAGING]
-host = localhost, localhost
-hostid = localhost1, localhost2
-channel = joinmarket-pit, joinmarket-pit
-port = 6667, 6668
-usessl = false, false
-socks5 = false, false
-socks5_host = localhost, localhost
-socks5_port = 9150, 9150
+[MESSAGING:server1]
```

Comment:
Is this change simply cosmetic?

Reply:
Mostly, I kept getting warnings about it being and "Old IRC configuration" and so updated the configuration file using the new format to silence the warning.

Reply:
Seems reasonable.

Reply:
Yes, this is fine (should have done it before really but it's fairly minor).